Can't build using build.ps1

On windows 10 w/ docker, running `build.ps1` makes it pretty far and then fails on ` webpack --config app-config/webpack.run.prod.js --display-error-details --bail`:


```
ModuleNotFoundError: Module not found: Error: Can't resolve './../$$_gendir/app/app.module.ngfactory' in '/src/Squidex/app'
    at factoryCallback (/src/Squidex/node_modules/webpack/lib/Compilation.js:264:39)
    at factory (/src/Squidex/node_modules/webpack/lib/NormalModuleFactory.js:247:20)
    at resolver (/src/Squidex/node_modules/webpack/lib/NormalModuleFactory.js:65:21)
    at asyncLib.parallel.e (/src/Squidex/node_modules/webpack/lib/NormalModuleFactory.js:138:21)
    at /src/Squidex/node_modules/async/dist/async.js:3861:9
    at /src/Squidex/node_modules/async/dist/async.js:421:16
    at iteratorCallback (/src/Squidex/node_modules/async/dist/async.js:996:13)
    at /src/Squidex/node_modules/async/dist/async.js:906:16
    at /src/Squidex/node_modules/async/dist/async.js:3858:13
    at resolvers.normal.resolve (/src/Squidex/node_modules/webpack/lib/NormalModuleFactory.js:130:23)
    at onError (/src/Squidex/node_modules/enhanced-resolve/lib/Resolver.js:65:10)
    at loggingCallbackWrapper (/src/Squidex/node_modules/enhanced-resolve/lib/createInnerCallback.js:31:19)
    at runAfter (/src/Squidex/node_modules/enhanced-resolve/lib/Resolver.js:158:4)
    at innerCallback (/src/Squidex/node_modules/enhanced-resolve/lib/Resolver.js:146:3)
    at loggingCallbackWrapper (/src/Squidex/node_modules/enhanced-resolve/lib/createInnerCallback.js:31:19)
    at next (/src/Squidex/node_modules/tapable/lib/Tapable.js:234:11)
    at /src/Squidex/node_modules/enhanced-resolve/lib/UnsafeCachePlugin.js:40:4
    at loggingCallbackWrapper (/src/Squidex/node_modules/enhanced-resolve/lib/createInnerCallback.js:31:19)
    at runAfter (/src/Squidex/node_modules/enhanced-resolve/lib/Resolver.js:158:4)
    at innerCallback (/src/Squidex/node_modules/enhanced-resolve/lib/Resolver.js:146:3)
    at loggingCallbackWrapper (/src/Squidex/node_modules/enhanced-resolve/lib/createInnerCallback.js:31:19)
    at next (/src/Squidex/node_modules/tapable/lib/Tapable.js:234:11)
    at innerCallback (/src/Squidex/node_modules/enhanced-resolve/lib/Resolver.js:144:11)
    at loggingCallbackWrapper (/src/Squidex/node_modules/enhanced-resolve/lib/createInnerCallback.js:31:19)
    at next (/src/Squidex/node_modules/tapable/lib/Tapable.js:231:35)
    at resolver.doResolve.createInnerCallback (/src/Squidex/node_modules/enhanced-resolve/lib/DescriptionFilePlugin.js:44:6)
    at loggingCallbackWrapper (/src/Squidex/node_modules/enhanced-resolve/lib/createInnerCallback.js:31:19)
    at afterInnerCallback (/src/Squidex/node_modules/enhanced-resolve/lib/Resolver.js:168:10)
    at loggingCallbackWrapper (/src/Squidex/node_modules/enhanced-resolve/lib/createInnerCallback.js:31:19)
    at next (/src/Squidex/node_modules/tapable/lib/Tapable.js:234:11)
resolve './../$$_gendir/app/app.module.ngfactory' in '/src/Squidex/app'
  using description file: /src/Squidex/package.json (relative path: ./app)
    Field 'browser' doesn't contain a valid alias configuration
  after using description file: /src/Squidex/package.json (relative path: ./app)
    using description file: /src/Squidex/package.json (relative path: ./$$_gendir/app/app.module.ngfactory)
      no extension
        Field 'browser' doesn't contain a valid alias configuration
        /src/Squidex/$$_gendir/app/app.module.ngfactory doesn't exist
      .js
        Field 'browser' doesn't contain a valid alias configuration
        /src/Squidex/$$_gendir/app/app.module.ngfactory.js doesn't exist
      .ts
        Field 'browser' doesn't contain a valid alias configuration
        /src/Squidex/$$_gendir/app/app.module.ngfactory.ts doesn't exist
      .css
        Field 'browser' doesn't contain a valid alias configuration
        /src/Squidex/$$_gendir/app/app.module.ngfactory.css doesn't exist
      .scss
        Field 'browser' doesn't contain a valid alias configuration
        /src/Squidex/$$_gendir/app/app.module.ngfactory.scss doesn't exist
      as directory
        /src/Squidex/$$_gendir/app/app.module.ngfactory doesn't exist

npm ERR! Linux 4.9.36-moby
npm ERR! argv "/usr/local/bin/node" "/usr/local/bin/npm" "run" "build"
npm ERR! node v6.11.1
npm ERR! npm  v3.10.10
npm ERR! code ELIFECYCLE
npm ERR! squidex@1.0.0 build: `webpack --config app-config/webpack.run.prod.js --display-error-details --bail`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the squidex@1.0.0 build script 'webpack --config app-config/webpack.run.prod.js --display-error-details --bail'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the squidex package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     webpack --config app-config/webpack.run.prod.js --display-error-details --bail
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs squidex
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls squidex
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /src/Squidex/npm-debug.log
The command '/bin/sh -c cp -a /tmp/node_modules /src/Squidex/  && cd /src/Squidex  && npm run test:coverage  && npm run build:copy  && npm run build' returned a non-zero code: 1
Unable to find image 'squidex-build-image:latest' locally
Error response from daemon: repository squidex-build-image not found: does not exist or no pull access
Error response from daemon: No such container: squidex-build-container
Error response from daemon: No such container: squidex-build-container
```

--- a/src/Squidex/package.json
+++ b/src/Squidex/package.json
@@ -44,7 +44,7 @@
   "devDependencies": {
     "@angular/compiler-cli": "4.3.0",
     "@angular/tsc-wrapped": "4.3.0",
-    "@ngtools/webpack": "1.5.1",
+    "@ngtools/webpack": "1.5.3",
     "@types/core-js": "0.9.35",
     "@types/jasmine": "2.5.43",
     "@types/mousetrap": "1.5.33",

